src/Server/config.ts

```diff
@@ -56,7 +56,7 @@ export const ENABLE_NEW_AUCTIONS_FILTER: any = false
 export const ENABLE_QUERY_BATCHING: any = false
 export const ENABLE_SAVED_SEARCH: any = false
 export const ENABLE_GRAPHQL_CACHE: any = false
-export const ENABLE_GRAPHQL_PROXY: any = true
+export const ENABLE_GRAPHQL_PROXY: any = false
```

Comment:
This allows [the documented command for disabling the cache](https://github.com/artsy/force/blob/main/docs/caching.md#caching) to work correctly. Deployed environments all have explicit values set. I've updated our `.env.shared` with `ENABLE_GRAPHQL_PROXY=true` so that development environments continue to proxy.